src/App.tsx

```diff
@@ -1,16 +1,41 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import { ProductsListComponent } from '@/components/productsList/ProductsList.component.tsx';
 
 import { AboutComponent } from './components/about/About.component.tsx';
 import { FooterComponent } from './components/footer/Footer.component.tsx';
 import { HeaderComponent } from './components/header/Header.component.tsx';
 
 import './App.css';
 
+export type Page = 'about' | 'products';
+
 function App() {
+    const [products, setProducts] = useState([]);
```

Comment:
do not forget to use the power of typescript
```suggestion
    const [products, setProducts] = useState<Product[]>([]);
```